src/index.ts

```diff
@@ -0,0 +1,24 @@
+///<reference path="../typings/index.d.ts" />
+import { EscapeStack } from './EscapeStack';
+import { handleEscapeKeydown } from './domEventHandler';
+
+let globalStack;
+
+const es = new EscapeStack();
+handleEscapeKeydown(document.body, es.pop);
```

Comment:
It doesn't look like this is ever actually used -- was it just leftover from before `createEscapeStack` was implemented here?